src/herder/TxSetUtils.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#pragma once
+
+#include "util/UnorderedMap.h"
+#include "xdr/Stellar-types.h"
+#include <deque>
+#include <herder/TxSetFrame.h>
+
+namespace stellar
+{
+namespace TxSetUtils
+{
+
+UnorderedMap<AccountID, TxSetFrame::AccountTransactionQueue>
+buildAccountTxQueues(TxSetFrame const& txSet);
+
+TxSetFramePtr surgePricingFilter(Application& app, TxSetFrameConstPtr txSetPtr);
```

Comment:
nit: Can we make these functions a bit more consistent:
- Consistent `txSet` argument placement (e.g. the first argument)
- Consistent type of `txSet`. I'd use const reference if possible, but const ptr is also ok (due to some optimizations it enables)